README.md

```diff
@@ -794,7 +794,7 @@ has another meaning in the context of mutual funds.)
 🄳 Two important classes of stock are
 [**common stock**](https://en.wikipedia.org/wiki/Common_stock) and
 [**preferred stock**](https://en.wikipedia.org/wiki/Preferred_stock). In general, preferred stock
-has “[rights, preferences, and privileges](https://www.americanbar.org/publications/blt/2014/01/04_bigler.html)"
+has “[rights, preferences, and privileges](https://www.americanbar.org/publications/blt/2014/01/04_bigler.html)”
```

Comment:
Yes! This is the one I mentioned--Flowmark wouldn't fix it, didn't know what to do! Thanks!